rktlet/runtime/helpers_test.go

```diff
@@ -1,5 +1,5 @@
 /*
-Copyright 2016 The Kubernetes Authors.
+Copyright 2016-2017 The Kubernetes Authors.
```

Comment:
while you are at it, you could do the same change in other files that you touch in this PR